implementations/java-jsonschema-validator/bowtieJsonSchemaValidator.java

```diff
@@ -0,0 +1,215 @@
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.ValidationMessage;
+import java.io.*;
+import java.util.List;
+import java.util.Map;
```

Comment:
This import is unused

Reply:
Does the linter not catch unused imports + variables? If not can you suggest how we configure it to? Or are we not running one, I forget.

Reply:
It seems there is no java linter supported in pre-commit? I'm not familiar with this tool tbh. Usually you would use `sonarlint` or `checkstyle` with java

Reply:
https://github.com/gherynos/pre-commit-java it looks like provides a hook for `checkstyle` at least.

Reply:
nice - I could add it later if you don't think it's not strictly needed in this PR

Reply:
Yep fine with me